cmd/crictl/container.go

```diff
@@ -592,3 +594,12 @@ func ListContainers(client pb.RuntimeServiceClient, opts listOptions) error {
 	w.Flush()
 	return nil
 }
+
+func getImageRef(c *pb.Container) string {
```

Comment:
Actually in current CRI implementation `c.Image.Image` should mostly be image tag, I don't think you want to truncate it.
See:
* https://github.com/kubernetes-incubator/cri-containerd/blob/master/pkg/server/container_status.go#L49
* https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/dockershim/docker_container.go#L375


I think we could show the `c.Image.Image` directly.

Reply:
OK, will do.

Reply:
See: https://github.com/kubernetes/kubernetes/blob/08781adf13491e08ac25e7545a9099d8be73a9dd/pkg/kubelet/apis/cri/v1alpha1/runtime/api.proto#L443-L448

Still show `c.Image.Image` directly if `c.Image.Image` is imageID?